tests/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ else()
   cmake_policy(VERSION 3.26)
 endif()
 
+# Include Python headers from FindPython
+include_directories(${Python_INCLUDE_DIRS})
```

Comment:
This should happen via targets already. You shouldn't need to use targetless commands like this in modern CMake.